app/templates/main/error.hbs

```diff
@@ -1,7 +0,0 @@
-{{#if slug}}
```

Comment:
This should be rendered when a repository is not found, for example: https://travis-ci.org/foobarbaz/barbazfoo. The way it worked was that we set the `slug` here: https://github.com/travis-ci/travis-web/blob/master/app/routes/repo.js#L55-L63 and then `main/error` would be rendered by default when an error is thrown in the routes. So I think that the proper change here would be to move this code to `app/templates/error.hbs` (if `slug` is present render "repo not found", otherwise render whatever we're rendering there now)